library/core/src/array/mod.rs

```diff
@@ -344,7 +344,15 @@ impl<T: PartialOrd, const N: usize> PartialOrd for [T; N] {
     }
 }
 
-/// Implements comparison of arrays lexicographically.
+/// Implements comparison of arrays [lexicographically](https://en.wikipedia.org/wiki/Lexicographic_order).
+///
+/// Lexicographical comparison is a operation with the following properties:
+///  - Two ranges are compared element by element.
+///  - The first mismatching element defines which range is lexicographically less or greater than the other.
+///  - If one range is a prefix of another, the shorter range is lexicographically less than the other.
+///  - If two ranges have equivalent elements and are of the same length, then the ranges are lexicographically equal.
+///  - An empty range is lexicographically less than any non-empty range.
+///  - Two empty ranges are lexicographically equal.
```

Comment:
Does it make sense to duplicate this everywhere? I think it would be better to have it in one place, whether that's the wikipedia article or a trait-level section on `Ord`.

Reply:
How about now?

Reply:
Sorry, I was unclear - I think it makes more sense to have the *link* everywhere, but not to duplicate the text. I was imagining something like the way all the `Vec` docs link to 'capacity and reallocation': https://doc.rust-lang.org/std/vec/struct.Vec.html#method.with_capacity, https://doc.rust-lang.org/std/vec/struct.Vec.html#capacity-and-reallocation

Reply:
Add a link everywhere.

Reply:
PTAL~